lib/src/widgets/keys.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
 
 const kPropertiesIcon = Key('properties_icon');
+const kActionsIcon = Key('actions_icon');
+const kActionsCloseIcon = Key('actions_close_icon');
```

Comment:
Not related to this PR, but something to think about 
https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters